src/Microsoft.DocAsCode.MarkdownLite/JsonRenderer.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.MarkdownLite
+{
+    using System.Collections.Immutable;
+
+    public class JsonRenderer
+    {
+        public enum Type
```

Comment:
>public [](start = 8, length = 6)

is this enum also be private? and considering move private methods after public methos and into one region?